inspirehep/dojson/hep/fields/bd1xx.py

```diff
@@ -141,6 +146,33 @@ def _get_record(value):
             if x_value and x_value.isdigit():
                 return get_record_ref(x_value, 'authors')
 
+        def _get_contributor_role(value):
+            if not value:
+                return None
+
+            if not isinstance(value, str):
+                value = set(value)
+
+            values = force_force_list(value)
+
+            contributor_roles = []
+            for value in values:
+                if value in current_app.config['INSPIRE_LEGACY_ROLES']['editing']:
```

Comment:
What about making it more robust and force `value` and all the content of `INSPIRE_LEGACY_ROLES` to be lowercase, so that we are future proof.
Invenio 1 was case insensitive.